content/docs/tech-insights/track-datadog-slos/index.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Track Datadog SLOs
+publishedDate: '2023-07-25T18:15:00.0Z'
+description: How to track and visualize Datadog SLOs with Roadie Tech Insights
+---
+
+As a platform team it is important to measure the reliability and performance of services, and Datadog provides powerful tools to monitor and track them. Creating and monitoring Datadog Service Level Objectives (SLOs) is one way to do it. Defining SLOs across multiple components can be an objective as well, and with Tech Insights your team is now able to manage these.
+
+### **Prerequisites**
+
+Before you begin, make sure you have the following:
+
+- A Datadog account: Sign up for a Datadog account if you don't have one already.
+- Access to your system's components: Ensure you have the necessary permissions to access the components you want to track.
```

Comment:
It's not totally clear to me what this means. Do I need access to the Backstage components, or the Datadog "services"? Maybe explain why we're going to need the access. That will make it more clear.